nix/gecko_env.nix

```diff
@@ -75,6 +75,7 @@ in gecko.overrideDerivation (old: {
     mozconfig=$out/conf/mozconfig
     echo > $mozconfig "
     ac_add_options --enable-clang-plugin
+    ac_add_options --enable-debug
     ac_add_options --with-clang-path=${clang_4}/bin/clang
```

Comment:
Side question, why _4?

Reply:
Because of this: https://github.com/mozilla/release-services/blob/8eb84aa8f9f7db9dcff0af889ffc33d8b836edec/nix/gecko_env.nix#L5
I wonder if we can't update this inside nix.

Reply:
I think we can, but are we actually using this option or can we just remove it?